api/metadata_endpoints.py

```diff
@@ -56,14 +60,34 @@ def datetime_to_iso_string(value: datetime) -> str:
 
 def get_landing_page(request):
     return LandingPageModel(
-        title="E-SOH EDR API",
-        description="The E-SOH EDR API",
-        keywords=["weather", "temperature", "wind", "humidity", "pressure", "clouds", "radiation"],
+        title="Observations Europe EUMETNET",
+        description=(
+            "EDR data service for European environmental observations from EUMETNET," " co-funded by the European Union"
```

Comment:
I wasn't part of this discussion, but I think it should be in the title that it is an EDR API.

Also, why is it now called an "EDR data service", and not an EDR API?
I would even refer to OGC EDR API here.